### PR TITLE
adds modernizing mlab blog post, and associated blog index pages

### PR DESCRIPTION
**Pull request by critzo**
This PR adds a blog post discussing the platform modernization project. 
Content can be previewed on my fork: https://critzo.github.io/m-lab.github.io/blog/modernizing-mlab/

Please note that external site links do not resolve properly on github hosted pages for forks, but resolve correctly when pushed to our live site.

@georgiamoon PTAL?

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/341)

### CODE REVIEW

**Comment by georgiamoon** [comment]
Review status: 0 of 7 files reviewed at latest revision, 5 unresolved discussions.

---

*[_posts/blog/2018-01-24-modernizing-mlab.md, line 18 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UuA3diHH5DQsYauH6:-L3UuA3diHH5DQsYauH7:bagkls6) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L18)):*
> In 2016
Should this be 2017?

---

*[_posts/blog/2018-01-24-modernizing-mlab.md, line 23 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UuSW8m6KExzoI3mC6:-L3UuSW8m6KExzoI3mC7:b-925esw) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L23)):*
> Each M-Lab pod has between 1-10 Gbps upstream connectivity, except in some regions of the world where the internet is less developed.

I would cut 'except in some regions of the world where the internet is less developed.' since that's not always the reason why we have less than 10, and right now the sentence reads that we might have less than 1 Gbps also, which isn't true.

---

*[_posts/blog/2018-01-24-modernizing-mlab.md, line 27 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uv037KHOxHPWDyHpE:-L3Uv037KHOxHPWDyHpF:b-wo4m9v) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L27)):*
> However, Planet Lab was designed and built before the community
This is the first mention of Planet Lab, we should probably clarify in the paragraph before this that the M-Lab kernel is from Planet Lab or built in collaboration with them, however best to explain it. 

---

*[_posts/blog/2018-01-24-modernizing-mlab.md, line 45 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uvf5B2nZtIfaFYP2I:-L3Uvf5B2nZtIfaFYP2J:bihw12t) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L45)):*
> beginning in late fall 2017.
Since late 2017 has now passed, we should update this date. 'begin in Spring 2018' or 'began in Fall 2017, and will happen throughout 2018'

---

*[_posts/blog/2018-01-24-modernizing-mlab.md, line 49 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uw2Vep_01k6mcYpJ5:-L3Uw2Vep_01k6mcYpJ6:b109e0n) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L49)):*
> support ASN annotation in a few months
Maybe say this year?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 27 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uv037KHOxHPWDyHpE:-L3Y2IgkpoA5gNIbcBj_:b-gb3fnc) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L27)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

> However, Planet Lab was designed and built before the community
This is the first mention of Planet Lab, we should probably clarify in the paragraph before this that the M-Lab kernel is from Planet Lab or built in collaboration with them, however best to explain it. 
</blockquote></details>

Agreed. Adding a sentence about PLC.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by georgiamoon** [comment]
tagging @pboothe to review the PL pieces and make sure this represents that correctly!

**Comment by critzo** [comment]
@georgiamoon I just pushed a commit to address your comments. PTAL?
@pboothe If you could please review the paragraph re: PLC that would be awesome.

File: _posts/blog/2018-01-24-modernizing-mlab.md [hunk 1]
```diff
@@ -9,18 +9,21 @@ categories:
...
+All M-Lab servers today are hosted either in well-connected data centers or within the research and education networks of contributing partner institutions. Each location has at least one "pod" which consists of 3-4 servers and one switch, connected directly to an upstream transit provider. Each M-Lab pod has between 1-10 Gbps upstream connectivity.
+
+The kernel, virtualization, and operating system running on M-Lab servers has historically been based on [PlanetLab](https://www.planet-lab.org/){:target="_blank"}. Since 2003, the PlanetLab Consortium has provided a research platform for researchers to prototype and develop, "new technologies for distributed storage, network mapping, peer-to-peer systems, distributed hash tables, and query processing." During this modernization project, M-Lab will remove most of our technical dependencies on PlanetLab. However, PlanetLab will continue to play an important role in the M-Lab Consortium.
```
Comment (pboothe) [inline _posts/blog/2018-01-24-modernizing-mlab.md]: Can we just delete this paragraph? I keep trying to rewrite it and not coming up with a good rewrite.

File: _posts/blog/2018-01-24-modernizing-mlab.md [hunk 2]
```diff
@@ -58,7 +61,7 @@ The design process and initial development of [ePoxy](https://github.com/m-lab/e
...
+To remove M-Lab’s reliance on Web100, M-Lab plans to transition to the TCP_INFO kernel instrumentation. Basing our measurements on TCP_INFO kernel instrumentation will allow M-Lab to support the latest version of TCP (called [BBR TCP](https://queue.acm.org/detail.cfm?id=3022184)){:target="_blank"} that will allow download tests to converge much more quickly at high speeds. M-Lab has confirmed that the current TCP_INFO implementation in modern kernels provides equivalent measurements of the TCP stack. Work is beginning now to refactor or turn down all M-Lab tools that use web100 to use TCP_INFO, including: NDT, Sidestream, NPAD, and the Paris-Traceroute wrapper. Once this work is completed, we’ll test these new tools on our platform’s testing infrastructure while keeping web100 on all production servers.
```
Comment (pboothe) [inline _posts/blog/2018-01-24-modernizing-mlab.md]: Change to `M-Lab plans to transition its hosted experiments to use the netlink socket and the TCP_INFO ....`

File: _posts/blog/2018-01-24-modernizing-mlab.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+### Remote Boot Management and Server Provisioning
+
+The design process and initial development of [ePoxy](https://github.com/m-lab/epoxy), a safe network bootloader and API, began in 2015. ePoxy is a replacement for the Planet Lab network bootloader used by M-Lab to load authenticated operating systems and virtualization containers securely across the Internet. Our development team has confirmed that ePoxy can be used to deploy an M-Lab server’s current operating system (Centos) and vserver virtualization containers. We also tested deployment of a container running [CoreOS](https://coreos.com/), which M-Lab intends to use as the base operating system on our servers after this platform upgrade is completed. The move to ePoxy will remove M-Lab’ dependency on Planet Lab and therefore addresses one reason that M-Lab remains on an older Linux kernel.
```
Comment (pboothe) [inline _posts/blog/2018-01-24-modernizing-mlab.md]: Please change to `The move to ePoxy will remove M-Lab's boot-time dependency on Planet Lab and therefore addresses one reason that M-Lab remains on an older Linux kernel. We anticipate continuing to use PlanetLab's identity-management systems on our updated platform.`

**Comment by georgiamoon** [comment]
One last note, some of the 

> {:target="_blank"}

formatting seems to be coming through in the blog post, might be a quotes issue?

File: _posts/blog/2018-01-24-modernizing-mlab.md (hunk 1 above)
Comment (critzo) [reply _posts/blog/2018-01-24-modernizing-mlab.md]: @pboothe Which paragraph?

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 18 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UuA3diHH5DQsYauH6:-L3cwS_VRMAFfq_UJ25G:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L18)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

> In 2016
Should this be 2017?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 23 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UuSW8m6KExzoI3mC6:-L3cwUdWhInYPgmtB_Xy:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L23)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

> Each M-Lab pod has between 1-10 Gbps upstream connectivity, except in some regions of the world where the internet is less developed.

I would cut 'except in some regions of the world where the internet is less developed.' since that's not always the reason why we have less than 10, and right now the sentence reads that we might have less than 1 Gbps also, which isn't true.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 26 at r3](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UhQonJfSVrOklXPN0-r3-26:-L3cwblu34W6Plz9BWwb:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/997e5f2af496c4a6a824de69dde09b3dec586714/_posts/blog/2018-01-24-modernizing-mlab.md#L26)):*
<details><summary><i>Previously, critzo wrote…</i></summary><blockquote>

@pboothe Which paragraph?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 26 at r3](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UhQonJfSVrOklXPN0-r3-26:-L3cwg1uHlvqeVBz5W2u:b5i0cxd) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/997e5f2af496c4a6a824de69dde09b3dec586714/_posts/blog/2018-01-24-modernizing-mlab.md#L26)):*
<details><summary><i>Previously, critzo wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Actually, we need something about who PLC is here, because we mention PLC in the list below it. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 26 at r3](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UhQonJfSVrOklXPN0-r3-26:-L3cxQeRKvZPBC3aEDtA:b-atmxkn) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/997e5f2af496c4a6a824de69dde09b3dec586714/_posts/blog/2018-01-24-modernizing-mlab.md#L26)):*
<details><summary><i>Previously, critzo wrote…</i></summary><blockquote>

Actually, we need something about who PLC is here, because we mention PLC in the list below it. 
</blockquote></details>

My next commit will keep a basic reference to PLC without some of the extra detail.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 27 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uv037KHOxHPWDyHpE:-L3cxXYlwUCZi2bZdzz9:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L27)):*
<details><summary><i>Previously, critzo wrote…</i></summary><blockquote>

Agreed. Adding a sentence about PLC.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 45 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uvf5B2nZtIfaFYP2I:-L3cxZS-XZaVt_JKrpfj:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L45)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

> beginning in late fall 2017.
Since late 2017 has now passed, we should update this date. 'begin in Spring 2018' or 'began in Fall 2017, and will happen throughout 2018'
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 49 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3Uw2Vep_01k6mcYpJ5:-L3cxa9ccrWUdTbd4Gxr:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L49)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary><blockquote>

> support ASN annotation in a few months
Maybe say this year?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 57 at r1](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UhQonJfSVrOklXPN0-r1-57:-L3cxu8OpLY-tS_NPEsx:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/2f8a9f7d1a5b614c867f85ebea32205beb476e3d/_posts/blog/2018-01-24-modernizing-mlab.md#L57)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

Please change to `The move to ePoxy will remove M-Lab's boot-time dependency on Planet Lab and therefore addresses one reason that M-Lab remains on an older Linux kernel. We anticipate continuing to use PlanetLab's identity-management systems on our updated platform.`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Comment by critzo** [comment]
*[_posts/blog/2018-01-24-modernizing-mlab.md, line 64 at r3](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341#-L3UhQonJfSVrOklXPN0-r3-64:-L3cy6po1OLVNXb2kqER:b-896fix) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/997e5f2af496c4a6a824de69dde09b3dec586714/_posts/blog/2018-01-24-modernizing-mlab.md#L64)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

Change to `M-Lab plans to transition its hosted experiments to use the netlink socket and the TCP_INFO ....`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/341)*

**Pull request merged**